product_docs/docs/biganimal/release/using_cluster/managing_replicas.mdx

```diff
@@ -0,0 +1,142 @@
+ 
+1.  The fields in the **Instance Type** section are populated with values inherited from the source cluster. You can edit these fields. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on instance type settings.
+
+1.  The fields in the **Storage** section are populated with values inherited from the source cluster. You can edit these fields. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on the storage settings.
+ 
+1.  In the **Networking** section, specify whether to use private or public networking. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on the Networking settings.
+
+1. On the DB Configuration tab, to avoid replication issues when running a replica (standby server), keep the inherited value or increase it for the following database configuration parameters. Otherwise, queries are not allowed in the replica. You can't edit any of the other parameters for replicas.
+   - max_connections
+
+   - max_locks_per_transaction
+   - max_prepared_transactions
+   - max_wal_senders
+   - max_worker_processes
+
+1. On the Additional Settings tab, in the **Backups** section, change the default replica backup retention period of 30 days using the **Retention Time** controls. You can configure the retention period as follows:
+   - 1 - 180 days
+
+   - 1 - 25 weeks
+   - 1 - 6 months
+
+1. Enable Identity and Access Management (IAM) Authentication to turn on the ability to log in to Postgres using your AWS IAM credentials. See [Authentication](/biganimal/latest/getting_started/creating_a_cluster/#authentication.
+
+1. Select **Create Replica**. 
+
+
+## Promote a faraway replica
+
+You can promote a faraway replica to a full-fledged cluster, which makes it capable of accepting writes. The new cluster created is different from the source cluster, and it doesn't replace the source cluster.
+
+1. Go to the [Clusters](https://portal.biganimal.com/clusters) page. A list of previously created clusters appears.
+
+1. Select the cluster with the replica you want to promote. In the **Overview** tab, you can see the cluster's replicas under **Faraway Replicas**.
+
+1. Select the **Promote Faraway Replica** icon next to the replica you want to promote. The Promote Faraway Replica page appears. 
+
+1. The cluster settings are populated with values inherited from the source cluster. You can edit the cluster settings while creating your cluster. 
+
+   !!! NoteNotes
+       - You can promote a faraway replica to a single node or high availability cluster but not to an extreme high availability cluster. 
+       - While promoting a replica to a cluster, you can't modify the **Provider**, **Database Type**, and **Region** fields or enable read-only workloads.
```

Comment:
We can add **Database Version** to the list as user can not modify it.

Reply:
Ah, **Database Type** include both **Postgres Type** and **Postgres Version** ?

Reply:
clarified a bit